src/IOExtras.jl

```diff
@@ -10,25 +10,37 @@ module IOExtras
 using ..Sockets
 using MbedTLS: MbedException
 
-export bytes, ByteView, nobytes, CodeUnits, IOError, isioerror,
+export bytes, isbytes, nbytes, ByteView, nobytes, IOError, isioerror,
```

Comment:
The changes here are a consolidation of the previous `bodybytes`/`bytes` and `bodylength` functions into a single place. There was some unfortunate mixups of when to use `bodybytes` vs `bytes` and we don't really need both.